doc/Contribution.md

```diff
@@ -0,0 +1,118 @@
+# Contribution Guidelines
+==================
+## Overview
+Being an Open Source project, everyone can contribute, provided that you respect the following points:
+* Before contributing any code, the author must make sure all the tests work (see below how to launch the tests).
```

Comment:
It's usually configured in all our repositories, but to use Travis to check your changes is quite slow, as you have to create the PR and wait for it to give you the red or green flag. It's easier for the developer to test that locally, so no nasty surprises will arise once the PR has been created. In that sense, Travis is more like a double-check for the PR reviewers than for the developer itself, IMHO.